lib/Cake/Cache/Cache.php

```diff
@@ -495,6 +495,9 @@ public static function clearGroup($group, $config = 'default') {
  */
 	public static function isInitialized($config = 'default') {
 		if (Configure::read('Cache.disable')) {
+			if (Configure::read('debug') > 1) {
```

Comment:
IMO `if (Configure::read('debug'))` suffices as it should only be boolean from here on (0 or not 0), 1 is never really something that is supported in 2.x.